docs/conf.py

```diff
@@ -59,6 +59,8 @@
     sys.exit(1)
 
 from sphinx_astropy.conf.v1 import *  # noqa: E402, F403
+from sphinx_astropy.conf.v1 import (
```

Comment:
I must have missed this convo. tl;dr -- why the import star and then explicit import from the same place?

Reply:
https://github.com/astropy/astropy/pull/13853#discussion_r1002588599

Reply:
Basically, the explicit import is for F401, the star import is for everything else. We should be switching to explicit imports only, but that's a different convo — #13785

Reply:
OK, thanks for the clarification!